context.c

```diff
@@ -355,6 +355,17 @@ int iio_context_init(struct iio_context *ctx)
 int iio_context_get_version(const struct iio_context *ctx,
 		unsigned int *major, unsigned int *minor, char git_tag[8])
 {
+	if (ctx->git_tag) {
+		if (major)
+			*major = ctx->major;
+		if (minor)
+			*minor = ctx->minor;
+		if (git_tag)
+			iio_strlcpy(git_tag, ctx->git_tag, 8);
```

Comment:
`sizeof(git_tag)`?

Reply:
That would work, as long as you execute it on a 64-bit OS ;)

Try this:
```
#include <stdio.h>

static void test(char array[16]) {
	char array2[16];

	printf("Size 1: %u size2: %u\n", sizeof(array), sizeof(array2));
}

int main(void)
{
	test(NULL);
	return 0;
}
```

Reply:
Hmm, I see. I was actually mistaken by the argument which I'm not really a fan... I see that we are just doing the size of the pointer :sweat_smile:

Reply:
Yes, it's a terrible design mistake I made when writing the old code. I'll update that in the 1.x libiio.